index.html

```diff
@@ -18202,11 +18209,16 @@ <h2 id="AudioWorkletGlobalScope">
                       </li>
                     </ul>
                   </li>
-                  <li>d the key-value pair (<em>name</em> -
+                  <li>Add the key-value pair (<em>name</em> -
                   <em>definition</em>) to the <a>node name to processor
                   definition map</a> of the associated
                   <a>AudioWorkletGlobalScope</a>.
                   </li>
+                  <li>Queue a task to the <a>control thread</a> to add the
+                  key-value pair (<em>name</em> - <em>descriptors</em>) to the
+                  <a>node name to parameter descriptors map</a> of the
```

Comment:
This link isn't working. I think you want "descriptor", not "descriptors".

Reply:
These are variables, not links to a definition, and the _descriptors_ variable is already defined above. I would have used a `<var>` tag but `<em>` was already used by existing language in this section.

Reply:
Sorry, I meant `<a>node name to parameter descriptors map</a>` is not linked. I think it should be "descriptor".